### PR TITLE
Update: 49fb660...92e4e19

### PR DESCRIPTION
**Pull request by fralonra**
https://github.com/fastify/fastify/compare/49fb660...92e4e19
  - https://github.com/fastify/fastify/pull/2817
  - https://github.com/fastify/fastify/pull/3192
  - https://github.com/fastify/fastify/pull/3198
  - https://github.com/fastify/fastify/pull/3201
  - https://github.com/fastify/fastify/pull/3203
  - https://github.com/fastify/fastify/pull/3205
  - https://github.com/fastify/fastify/pull/3210
  - https://github.com/fastify/fastify/pull/3213
  - https://github.com/fastify/fastify/pull/3222
  - https://github.com/fastify/fastify/pull/3223
  - https://github.com/fastify/fastify/pull/3227
  - https://github.com/fastify/fastify/pull/3241
  - https://github.com/fastify/fastify/pull/3248

### CODE REVIEW

File: docs/ContentTypeParser.md [hunk 1]
```diff
@@ -66,6 +70,36 @@ if (!fastify.hasContentTypeParser('application/jsoff')){
...
+  })
+})
+
+// 移除内建的 content type 解析器。这时只有上文添加的 text/html 解析器可用。
+fastiy.removeContentTypeParser(['application/json', 'text/plain'])
+```
+
+#### removeAllContentTypeParsers
+
+在上文的例子中，你需要明确指定所有你想移除的 content type。但你也可以使用 `removeAllContentTypeParsers`直接移除所有现存的 content type 解析器。在下面的例子里，我们实现了一样的效果，但不再需要手动指定 content type 了。和 `removeContentTypeParser` 一样，该 API 也支持封装。当你想注册一个[能捕获所有 content type 的解析器](#Catch-All)，且忽略内建的解析器时，你可使用 `removeAllContentTypeParsers`。
```
Comment (vincent178) [inline docs/ContentTypeParser.md]: > The API is especially useful if you want to register a 
[catch-all content type parser](#Catch-All) that should be executed for every content type and the built-in parsers should be ignored as well.

当你想注册一个[能捕获所有 content type 的解析器](#Catch-All)，且忽略内建的解析器时，这个 API 特别有用。

File: docs/ContentTypeParser.md [hunk 2]
```diff
@@ -130,4 +174,20 @@ fastify.route({
   }
 })
 ```
-关于输送上传的文件，请看[该插件](https://github.com/fastify/fastify-multipart)。
\ No newline at end of file
+
+关于输送上传的文件，请看[该插件](https://github.com/fastify/fastify-multipart)。
```
Comment (vincent178) [inline docs/ContentTypeParser.md]: `关于输送上传的文件` 是不是改成 `关于处理上传的文件`

**Review by vincent178 — APPROVED** [review]
LGTM, some minor comments/suggestion.

File: docs/ContentTypeParser.md (hunk 2 above)
Comment (fralonra) [reply docs/ContentTypeParser.md]: LGTM

**Pull request merged**